Modules/Devices/test/Devices.Domain.Tests/Identities/DeletionProcessApprovalReminderTests.cs

```diff
@@ -103,6 +103,20 @@ public void DeletionProcessApprovalReminder3Sent_fails_when_no_deletion_process_
         acting.Should().Throw<DomainException>().Which.Code.Should().Be("error.platform.validation.device.noDeletionProcessWithRequiredStatusExists");
     }
 
+    [Fact]
+    public void GetEndOfApprovalPeriod_returns_expected_date()
+    {
+        // Arrange
+        SetupSystemTime();
```

Comment:
I really don't like this method. You now always have to navigate into this method if you want to see what's UtcNow.

What's so bad about the following?

```c#
SystemTime.Set(DateTime.Parse("2000-01-01"));
```

Reply:
Uhm, this was actually a suggestion you had for the GracePeriod tests, so I used it here since it's the exact same context.

Reply:
Or do you mean for this specific test?

Reply:
Did I? 😮 If so, that is really embarrissing. 😆

Reply:
Shall I change it?

Reply:
> Or do you mean for this specific test?

Well, for this one in particular. But I can't think of ANY test where it is good. Can you send me the comment in which I suggested this?

Reply:
> Shall I change it?

That would be great